src/ds-style-fix.scss

```diff
@@ -159,6 +159,31 @@ table.hosted-workflow-versions-grid {
   margin-bottom: 10px;
 }
 
+table.hosted-tool-versions-grid {
```

Comment:
col 2 4 2 2 2 not good enough?

Reply:
Was just maintaining the approach we currently had